javascript/javascript3/week1/Weather App/wearther.js

```diff
@@ -0,0 +1,50 @@
+function weatherOfCurrentCity () {
+   
+    var nameCity = document.getElementById("nameCity").value;
+    if(nameCity.trim()=== "")  {
+        alert("Kindly write the city name!");

```

Comment:
`alert` is bad practice and deprecated. Please use a error message that is rendered into html